internal/manifests/naming/naming.go

```diff
@@ -5,11 +5,24 @@ import (
 )
 
 // Name returns the manifest name of a component.
-func Name(component string, instanceName string) string {
+// Example: tempo-simplest-compactor.
+func Name(component string, tempoStackName string) string {
 	if component == "" {
-		return fmt.Sprintf("tempo-%s", instanceName)
+		return fmt.Sprintf("tempo-%s", tempoStackName)
 	}
-	return fmt.Sprintf("tempo-%s-%s", instanceName, component)
+	return fmt.Sprintf("tempo-%s-%s", tempoStackName, component)
+}
+
+// ServiceName returns the name of a service of a component.
+// Example: tempo-simplest-compactor-http
+// Note: This is not the name of the Kubernetes service object.
```

Comment:
Maybe, since you added a note explaining what this method does not return, it would be nice to:
* Point to the method that return the name of the Kubernetes object
* Explain for what kind of stuff is this method used.

Reply:
Done